spec/support/purl_helpers.rb

```diff
@@ -14,6 +14,16 @@ def expect_text_on_purl_page(druid:, text:, within_frame: false)
     end
   end
 
+  def do_not_expect_text_on_purl_page(druid:, text:, within_frame: false)
```

Comment:
I'm curious about the `within_frame` arg here. Does anything want that to be `true`?